src/SignUpIn/DonatingAgency/DonatingAgencySignUpController.js

```diff
@@ -60,6 +62,62 @@ class DonatingAgencySignUpController extends Component {
         });
     }
 
+    submitRegistration() {
+        // Handle via ajax submitting the user data, upon
+        // success return this.nextStop(). If it fails,
+        // show the user the error but don't advance
+
+        firebase.auth().createUserWithEmailAndPassword(fieldValues.adminEmail, fieldValues.adminPassword)
+            .then(user => {    
+                let agencyKey = firebase.database().ref().child('accounts').push().key;
+                let member_postData = {
+                    accountType: 'donating_agency_member',
+                    agency: agencyKey,
+                    name: fieldValues.memberName,
+                    email: fieldValues.memberEmail,
+                    phone: fieldValues.memberPhone,
+                    position: fieldValues.memberPosition,
+                    isAdmin: true,
+                    notification: {
+                        type: NotificationType,
+                        content: '-L5QoXeC_UrL5tRRED3e'
+                    }
+                };
+
+                let agency_postData = {
+                    school: 'RheaQY1WxJT03sTPQICFZ4STpfm1',
```

Comment:
same, `umbrella`, remember to change to the new one

Reply:
Fixed.